analyzer/src/main/kotlin/Main.kt

```diff
@@ -177,39 +171,19 @@ object Main {
         println("The following package managers are activated:")
         println("\t" + packageManagers.joinToString(", "))
 
-        // Map of paths managed by the respective package manager.
-        val managedProjectPaths = mutableMapOf<PackageManagerFactory<PackageManager>, MutableList<File>>()
-
         val absoluteProjectPath = inputDir.absoluteFile
         println("Scanning project path:\n\t$absoluteProjectPath")
 
-        if (packageManagers.size == 1 && absoluteProjectPath.isFile) {
-            // If only one package manager is activated, treat given paths to files as definition files for that
-            // package manager despite their name.
-            managedProjectPaths.getOrPut(packageManagers.first()) { mutableListOf() }.add(absoluteProjectPath)
+        // Map of files managed by the respective package manager.
+        val managedProjectFiles = if (packageManagers.size == 1 && absoluteProjectPath.isFile) {
```

Comment:
I'm a bit unhappy about the `managedProjectFiles` name here. Inside a single package manager we call these 'definitionFiles'. As this maps package managers to their definition files, maybe call it `managedDefinitionFiles`? Or just `packageManagerFiles`?

Reply:
Went for managedDefinitionFiles.